draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -842,6 +842,19 @@ ULP:  Upper Layer Protocol
    user messages that are buffered in the SCTP layer MUST be read and
    processed by DTLS.
 
+   The interaction between peers and protocol stacks shall be as follows:
+1. Local peer A asks for terminating the DTLS/SCTP Association
+2. Local DTLS/SCTP acknowledge the request, from this time on no new messages will be accepted from A
+3. Local DTLS/SCTP flushes all data towards the remote peer B
+4. Local DTLS/SCTP waits until all data have been transferred to B via SCTP
+5. Local DTLS/SCTP sends Close_notify to remote peer B on each and all DTLS connections
```

Comment:
To me this transmission can happen as soon 3 has completed and the data is in the SCTP tx_buffer?